napari/resources/__init__.py

```diff
@@ -65,6 +65,7 @@ def import_resources(version: str = '', overwrite: bool = False) -> str:
         Version string, by default the resources will be written to a file that
         encodes the current napari version, as well as Qt backend and version:
         ``_qt_resources_{napari.__version__}_{API}_{QT_VERSION}.py``
+
```

Comment:
The heuristic is : if one section have one blankline somewhere we'll separate items with blanklines.